apps/web/src/pages/_app.tsx

```diff
@@ -3,13 +3,45 @@ import '../styles.css';
 import Loading from '@components/Shared/Loading';
 import circluarStd from '@lib/lensterFont';
 import type { AppProps } from 'next/app';
-import { lazy, Suspense } from 'react';
+import Head from 'next/head';
+import { lazy, Suspense, useMemo } from 'react';
+import { useAppPersistStore } from 'src/store/app';
 
 const Providers = lazy(() => import('@components/Common/Providers'));
 
 const App = ({ Component, pageProps }: AppProps) => {
+  const notificationCount = useAppPersistStore(
+    (state) => state.notificationCount
+  );
+
+  const hasNotifications = useMemo(
+    () => notificationCount > 0,
+    [notificationCount]
+  );
+
   return (
     <Suspense fallback={<Loading />}>
+      <Head>
+        <meta name="testme" content="yes" />
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href="/apple-touch-icon.png"
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="32x32"
+          href={`/favicon-32x32${hasNotifications ? '-notification' : ''}.png`}
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="16x16"
+          href={`/favicon-16x16${hasNotifications ? '-notification' : ''}.png`}
+        />
+      </Head>
```

Comment:
Maybe we can move to a separate component called `Favicon.tsx` and consume it in `Layout.tsx`

Reply:
Good idea. I've just moved it to a separate component. However, I couldn't keep it in Layout.tsx since it broke SSR which is vital for favicon. So it's still in _app.tsx